pkg/devcontainer/config/metadata.go

```diff
@@ -12,3 +12,11 @@ type ImageMetadata struct {
 	DevContainerActions    `json:",inline"`
 	NonComposeBase         `json:",inline"`
 }
+
+func AddConfigToImageMetadata(config *DevContainerConfig, imageMetadataConfig *ImageMetadataConfig) {
```

Comment:
nit: docblock for a public function please explaining what is does :pray:

Reply:
I added it but note that I don't see a lot of doc on other public functions in the project.

Reply:
@aacebedo you are completely right and I'm trying to change that :) https://tip.golang.org/doc/comment